docs/docs/unreachable.md

```diff
@@ -0,0 +1,198 @@
+- `--max_agent_ping_timeouts` (default 5) - The number of consecutive attempts to ping an agent, after which the agent is considered unreachable.
+
+With the Mesos defaults, an agent will be marked inactive after a 75 seconds (15 seconds \* 5) of no communication,
+assuming only 1 node becoming inactive.
+
+In DC/OS, the default for `--max_slave_ping_timeouts` is 20 [See dcos-config.yaml](https://github.com/dcos/dcos/blob/9cc6ab28060545cd203c09aa7fa1b9456773d080/gen/dcos-config.yaml#L449).
+As such, the Mesos master will consider an agent inactive after 5 minutes (20 \* 15 seconds) of no communication.  Again
+assuming only 1 node becoming inactive.
+
+When Mesos marks an agent as inactive and the agent isn't rate limited, Mesos will publish an unreachable task status
+update for all tasks associated with that agent.   Where things can be surprising is when we consider the agent rate limiter.
+
+
+### Agent Removal Rate Limiter
+
+In the previous section, we mentioned that Mesos will consider an agent inactive, assuming that only 1 agent is lost.
+This is because Apache Mesos has an agent rate limiter which was established prior to partition aware frameworks
+and is still enforced. The purpose of the rate limiter is to reduce the number of reported lost agents within a specified
+amount of time through the `--agent_removal_rate_limit` flag.
+
+In Mesos, the default for `--agent_removal_rate_limit` is `None`, which has the effect of reporting agents immediately
+after the agent health check logic as described above.
+
+The default in DC/OS however is 1/20mins [See dcos-config.yaml](https://github.com/dcos/dcos/blob/9cc6ab28060545cd203c09aa7fa1b9456773d080/gen/dcos-config.yaml#L442)
+which is 1 agent every 20 minutes. The following hypothetical timeline describes how 2 agents that become lost at the
+same time will be marked inactive:
+
+- 12:00:00 - Agent 1 goes off-line
+- 12:03:00 - Agent 2 goes off-line
+- 12:05:00 - Mesos Master marks agent 1 as inactive, and publishes a `TASK_UNREACHABLE` status update for relevant tasks.
+- 12:25:00 - Mesos Master marks agent 2 as inactive, and publishes a `TASK_UNREACHABLE` status update for relevant tasks.
+
+The net result is that the amount of time that a agent on DC/OS can be lost or unreachable before Marathon is notified is not always deterministic. It is defined by the number of lost nodes, and the order in which they were lost [^1]. It is important to understand that these tasks are listed by Mesos as Active until this time and Marathon is completely unaware until notified by Mesos.
+
+**NOTE:**
+There are 2 JIRAs against Mesos to remove rate limits. One marked as a critical bug ([MESOS-7721](https://issues.apache.org/jira/browse/MESOS-7721))
+the other as a major improvement ([MESOS-5948](https://issues.apache.org/jira/browse/MESOS-5948)). There is currently discussion in process to remove this rate limit default from DC/OS.
```

Comment:
Nit: "Currently" is relative to the time the doc is read.

I'd just remove the last sentence.